lib/foods.js

```diff
@@ -21,7 +26,7 @@ function checkInputs() {
   };
 };
 
-function loadButton() {
+function createDeleteButton() {
```

Comment:
@lucyconklin Nice, this would help to make the code better readable, easier to understanding what this function was meant to do.